charts/rd-professional-api/values.yaml

```diff
@@ -0,0 +1,29 @@
+rd-professional-api:
```

Comment:
This should be java unless you want to alias it in requirements.yaml to rd-professional-api

Reply:
```suggestion
java:
```

Reply:
Done